### PR TITLE
fix: hide extra blank space in dashboard chart menu

### PR DESCRIPTION
**Pull request by ktmud**
### SUMMARY

Hide extra blank space in the "Force Refresh" link in dashboard chart's dropdown menu, while chart query is still running.

#### Before

Not sure if it was by design originally, but it always kind of bothers me.

### TEST PLAN

Manual

1. Go to a dashboard with charts with very slow query
2. Click on "Force Refresh"

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [x] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/package-lock.json [hunk 1]
```diff
@@ -20532,7 +20532,8 @@
         },
         "kind-of": {
           "version": "6.0.2",
-          "resolved": "",
+          "resolved": "https://registry.npmjs.org/kind-of/-/kind-of-6.0.2.tgz",
```
Comment (etr2460) [inline superset-frontend/package-lock.json:20535]: are these changes intended?

File: superset-frontend/src/dashboard/components/SliceHeaderControls.jsx [hunk 2]
```diff
@@ -146,12 +146,14 @@ class SliceHeaderControls extends React.PureComponent {
...
-            <div
-              className="refresh-tooltip"
-              data-test="dashboard-slice-refresh-tooltip"
-            >
-              {refreshTooltip}
-            </div>
+            {refreshTooltip && (
+              <div
+                className="refresh-tooltip"
+                data-test="dashboard-slice-refresh-tooltip"
```
Comment (etr2460) [inline superset-frontend/src/dashboard/components/SliceHeaderControls.jsx:152]: seems like this could be really easy to add a unit test for since this already has a `data-test` attribute. Might be worth adding a test for when the dashboard is refreshing then the div isn't rendered

**Review by etr2460 — APPROVED** [review]
a couple comments, but nothing blocking

File: superset-frontend/package-lock.json (hunk 1 above)
Comment (ktmud) [reply superset-frontend/package-lock.json:20535]: Not really

File: superset-frontend/src/dashboard/components/SliceHeaderControls.jsx (hunk 2 above)
Comment (ktmud) [reply superset-frontend/src/dashboard/components/SliceHeaderControls.jsx:152]: will work on a test later

File: superset-frontend/package-lock.json (hunk 1 above)
Comment (rusackas) [reply superset-frontend/package-lock.json]: This stuff always seems to alternate back and forth. Not sure if there's a good way to reduce churn on this.

**Review by rusackas — APPROVED** [review]

**Comment by junlincc** [comment]
Confirm this is no longer an issue in master. thanks for the PR, ktmud. 🙏
